go.mod

```diff
@@ -114,3 +114,5 @@ require (
 )
 
 go 1.20
+
+replace github.com/codeready-toolchain/toolchain-common => github.com/filariow/toolchain-common v0.0.0-20240109160929-447985913c7f
```

Comment:
reminder

Reply:
thx

Reply:
We don't really have to use any reminders anymore. We have our `go.mod replacment` check which automatically blocks all PRs with `raplace` in it :)
See https://github.com/codeready-toolchain/host-operator/actions/runs/7463977391/job/20309889472?pr=955